sakura_core/util/StaticType.h

```diff
@@ -96,39 +96,39 @@ class StaticVector{
 
 //! ヒープを用いない文字列クラス
 //2007.09.23 kobake 作成。
-template <class CHAR_TYPE, int N_BUFFER_COUNT>
+template <int N_BUFFER_COUNT>
```

Comment:
指摘じゃありませんが、こうするのが良さそうです。

```suggestion
template <uint16_t N_BUFFER_COUNT, typename CHAR_TYPE = wchar_t>
```